web-ts/src/routes/RootSwitcher.tsx

```diff
@@ -30,23 +43,26 @@ import LoadingSpinner from '../components/LoadingSpinner'
 import { useTranslation } from 'react-i18next'
 
 export const cityContentPattern = `/:cityCode/:languageCode`
+type cityContentPatternType = typeof cityContentPattern
 
 export const RoutePatterns = {
-  [LANDING_ROUTE]: `/${LANDING_ROUTE}/:languageCode`,
-  [MAIN_DISCLAIMER_ROUTE]: `/${MAIN_DISCLAIMER_ROUTE}`,
-  [NOT_FOUND_ROUTE]: `/${NOT_FOUND_ROUTE}`,
+  [LANDING_ROUTE]: `/${LANDING_ROUTE}/:languageCode` as `/${LandingRouteType}/:languageCode`,
```

Comment:
The type of template strings are always just `string`. By adding types here we have type security in the components (i.e. which parameters are available for the route, e.g. `cityCode` or `eventId`).

Reply:
you could do the same as @SteffiStuffel did in her previous PR and declare RoutePatterns `as const` this way typescript is also able to correctly infer the corresponding types and all values would become readonly.

Reply:
:pray: thanks! Works like a charm! Applied in #312

Reply:
Very cool! :)